mail_operating_unit/models/mail_thread.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _message_compute_author(
+        self, author_id=None, email_from=None, raise_on_email=True
+    ):
+        alias_domain_id = self.env.context.get("record_alias_domain_id")
+        if alias_domain_id:
+            email_from = (
+                self.env["mail.alias.domain"].browse(alias_domain_id).default_from_email
```

Comment:
not super sure this is what we want. 

In my mind, I had more:

```
def _message_author(...):
    author_id, email_from = super()._message_author(...)
    alias_domain_id = self.env.context.get("record_alias_domain_id")
    if alias_domain_id:
        email_from = email_from.split('@')[0] + alias_domain_id.name
    return author_id, email_from 

```

Do you have an opinion on this?